DbgShell/ColorHostUserInterface.cs

```diff
@@ -662,13 +674,13 @@ internal void WriteToConsole(string value, bool transcribeResult)
 
             // This is atomic, so we don't lock here...
 
-            ConsoleControl.WriteConsole(handle, value);
+            ConsoleControl.WriteConsole(handle, value, newLine);
 
-            if (transcribeResult)
-            {
-                PostWrite(value);
-            }
-            else
+            //if (transcribeResult)
+            //{
+            //    PostWrite(value);
+            //}
+            //else
```

Comment:
Why neuter `transcribeResult` here? I don't feel like transcription is a very important feature for DbgShell, so if there were some reason we needed to kill it, I'd be fine with that, but we should at least have an explanation or something. But I suspect maybe this looks more like a case of commenting something out for whatever reason and then forgetting about it.